PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1146,16 +1146,15 @@ You can define several column markers by using white space to separate the diffe
                     <Item id="6824" name="Ignore numbers"/>
                     <Item id="6811" name="From"/>
                     <Item id="6813" name="th character"/>
-                    <Item id="6814" name="Valid value : 1 - 9"/>
                     <Item id="6872" name="Make auto-completion list brief"/>
                     <Item id="6815" name="Function parameters hint on input"/>
                     <Item id="6851" name="Auto-Insert"/>
                     <Item id="6857" name=" html/xml close tag"/>
                     <Item id="6858" name="Open"/>
                     <Item id="6859" name="Close"/>
-                    <Item id="6860" name="Matched pair 1:"/>
-                    <Item id="6863" name="Matched pair 2:"/>
-                    <Item id="6866" name="Matched pair 3:"/>
+                    <Item id="6860" name="Matched pair 1 :"/>
```

Comment:
Why we add space before ":"? Is it because of the visual improvement? I've checked several programs and none add spaces here. We currently have an open issue regarding this and I don't know what to do with it.
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/4628

Reply:
I did not know about that issue. I just wanted to keep it consistent.
IMHO I would not add space before `:`.